src/main/java/seedu/address/commons/core/CalendarManager.java

```
@@ -82,37 +82,57 @@
     private String calendarId;
     private ExecutorService executor;
 
-    private CalendarManager() throws IOException, GeneralSecurityException {
-
-        final NetHttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
-        if (CalendarManager.getSavedCredentials(httpTransport) != null) {
-            this.service = new Calendar.Builder(httpTransport, JSON_FACTORY,
-                    CalendarManager.getSavedCredentials(httpTransport))
-                    .setApplicationName(APPLICATION_NAME)
-                    .build();
-            String calendarId = getCalendarIdBySummary(CALENDAR_TITLE);
-            if (calendarId == null) {
-                createNewCalendar();
-                setCalendarAccess();
-            } else {
-                this.calendarId = calendarId;
+    private CalendarManager() {
+        try {
+            final NetHttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
+            if (CalendarManager.getSavedCredentials(httpTransport) != null) {
+                this.service = new Calendar.Builder(httpTransport, JSON_FACTORY,
+                        CalendarManager.getSavedCredentials(httpTransport))
+                        .setApplicationName(APPLICATION_NAME)
+                        .build();
+                initializeCalendar();
             }
+        } catch (IOException | GeneralSecurityException ex) {
+            logger.info("Error initializing calendar manager");
```

I think it would be better to propagate this exception upwards so that this message gets shown in the command box rather than in the logs.